<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>properties</class>
 <widget class="QWidget" name="properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>178</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="appNameLabel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="versionLabel">
     <property name="text">
      <string>&amp;Version number:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_appVersion</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="kcfg_appVersion">
     <property name="toolTip">
      <string>Project's version number</string>
     </property>
     <property name="whatsThis">
      <string>Set your project version number. A first project should start with version 0.1.</string>
     </property>
     <property name="inputMask">
      <string>0.0</string>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="installLabel">
     <property name="text">
      <string>In&amp;stallation directory:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_url</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KUrlRequester" name="kcfg_url">
     <property name="toolTip">
      <string>The directory where you will build your project</string>
     </property>
     <property name="whatsThis">
      <string>Choose the directory where you will build your project. Your home /src is a good default location.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>Author &amp;name:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_name</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KLineEdit" name="kcfg_name">
     <property name="toolTip">
      <string>Your first name and name</string>
     </property>
     <property name="whatsThis">
      <string>This will set the copyright to this name</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="emailLabel">
     <property name="text">
      <string>A&amp;uthor email:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_email</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KLineEdit" name="kcfg_email">
     <property name="toolTip">
      <string>Your email address</string>
     </property>
     <property name="whatsThis">
      <string>This email address will be next to your name in the copyright credit of the project files.</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
